api/types.go

```diff
@@ -609,7 +611,7 @@ func DefaultOptions() Options {
 
 		Runner: Runner{
 			// options set when the model is loaded
-			NumCtx:    2048,
+			NumCtx:    int(envconfig.ContextLength()),
```

Comment:
This looks great. It's nominal, but could we add a test to `types_test.go` that calls `t.Setenv("OLLAMA_CONTEXT_LENGTH", <value>)` and then compares the default options?

Reply:
yes that's a good idea!